src/ForgeVTT.mjs

```diff
@@ -61,36 +61,41 @@ export class ForgeVTT {
   }
 
   /**
-   * The global isNewerVersion will be removed in v14, so we need a utility function to alias to whichever is available.
-   * @param {string} version The version to check
+   * Determines if the current Foundry VTT version is newer than the specified target version.
+   *
    * @param {string} target The version to check against
-   * @returns {boolean} True when the version is newer than the target, false otherwise
-   * @todo This may be a duplicate of ForgeCompatibility.isNewerVersion
+   * @returns {boolean} True if the current Foundry VTT version is newer than the target version, otherwise false.
    */
-  static _isNewerVersion(version, target) {
-    try {
-      return foundry.utils.isNewerVersion(version, target);
-    } catch {
-      return isNewerVersion(version, target);
-    }
+  static isNewerFoundryVersion(target) {
```

Comment:
suggestion (non-blocking): rename to `isFoundryVersionAtLeast` or `isFoundryVersionNewerThan`

I never liked the name `isNewerFoundryVersion` or `isNewerVersion` as that name, at least for me does not work well (especially if we have only one argument). I think changing the name to one of the suggestions (depending on the logic inside) will make it obvious what the function does and what can you expect from the output.

Reply:
I agree, but I think it's equally important to match naming with Foundry's overriden or wrapped methods :thinking: here I think it is better to remind of `foundry.utils.isNewerVersion(version, target)`...

Let's hear what the gang thinks about it before I apply one of your suggestions :+1: